dependency-reduced-pom.xml

```diff
@@ -1,10 +1,35 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
-  <groupId>net.wedjaa.ansible.vault</groupId>
+  <groupId>co.actioniq</groupId>
   <artifactId>vault-utilities</artifactId>
-  <version>1.2.0</version>
+  <name>vault-utilities</name>
+  <version>1.2.1</version>

```

Comment:
Did you publish 1.2.1? So then this should be `1.2.2-SNAPSHOT` to match below?

Reply:
I don't think this file needs to even be checked in gonna remove it and git ignore it